.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -0,0 +1,26 @@
+### [Github issues](https://github.com/Microsoft/botframework-webchat/issues) should be used for bugs and feature requests. See the Support section to get support related to Bot Framework and Web Chat.
+
+## Screenshots
+<!-- If applicable, add screenshots to help explain your problem. -->
```

Comment:
Do you think we should give notes about personal-identifiable information should not be in screenshots?